### ISSUE TITLE
Hammer Commands test: BZ1219610 was closed WONTFIX

### ISSUE THREAD

**Issue by lpramuk** [SatelliteQE/robottelo#4180]
- [ ] tests.foreman.cli.test_hammer.HammerCommandsTestCase.test_positive_all_options

BZ https://bugzilla.redhat.com/show_bug.cgi?id=1219610 was CLOSED - WONTFIX on Jan 5 2017

See one occurrence here: 
https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/cli/test_hammer.py#L117-L120

Our automation consider ```CLOSED``` as ```FIXED``` !!!

We have to make the affected blocks permanent, otherwise the test will be failing like that:
```
$ nosetests -v test_hammer:HammerCommandsTestCase.test_positive_all_options
check all provided options for every hammer command ... FAIL

======================================================================
FAIL: check all provided options for every hammer command
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/lpramuk/git/SatelliteQE/robottelo/tests/foreman/cli/test_hammer.py", line 160, in test_positive_all_options
    '\n' + _format_commands_diff(self.differences)
AssertionError: 
hammer discovery_rule (new command)
  Added subcommands:
    * info
    * create
    * list
    * update
    * delete
  Added options:
    * help

hammer discovery_rule create (new command)
  Added options:
    * search
    * hostname
    * hostgroup-title
    * hostgroup-id
    * enabled
    * hostgroup
    * name
    * priority
    * hosts-limit
    * max-count
    * help

hammer discovery_rule delete (new command)
  Added options:
    * help
    * name
    * id

hammer discovery_rule info (new command)
  Added options:
    * help
    * name
    * id

hammer discovery_rule list (new command)
  Added options:
    * search
    * order
    * help
    * page
    * per-page

hammer discovery_rule update (new command)
  Added options:
    * search
    * hostname
    * hostgroup-title
    * hostgroup-id
    * enabled
    * hostgroup
    * name
    * priority
    * hosts-limit
    * max-count
    * id
    * new-name
    * help
```

**Comment by lpramuk** [comment]
Found when trying to submit test results for PR https://github.com/SatelliteQE/robottelo/pull/4166

**Comment by rochacbruno** [comment]
To be addressed in #4262

**Comment by rplevka** [comment]
closing in favor of https://github.com/SatelliteQE/robottelo/issues/4262
